tools/benchmark_tool/README.md

```diff
@@ -12,33 +13,43 @@ This topic demonstrates how to run the Benchmark Python* Tool, which performs in
 > Proceed to an [easy installation from Docker](@ref workbench_docs_Workbench_DG_Install_from_Docker_Hub) to get started.
 
 ## How It Works
-
-Upon start-up, the application reads command-line parameters and loads a network and images/binary files to the Inference Engine plugin, which is chosen depending on a specified device. The number of infer requests and execution approach depend on the mode defined with the `-api` command-line parameter.
+Upon start-up, the application reads command-line parameters and loads a network and inputs (images/binary files) to the specified device.
+Device-specific execution parameters (number of streams, threads, and so on) can be either explicitly specified through the command line
+or left default. In the latter case, the sample logic will select the values for the optimal throughput.
+While further experimenting with individual parameters (like number of streams and requests, batch size, etc) allows to find the performance sweet spot,
+usually, the resulting values are not very performance-portable,
+so the values from one machine or device are not necessarily optimal for another.
+From this perspective, the most portable way is experimenting only the performance hints. To learn more, refer to the section below.
 
 > **NOTE**: By default, Inference Engine samples, tools and demos expect input with BGR channels order. If you trained your model to work with RGB order, you need to manually rearrange the default channels order in the sample or demo application or reconvert your model using the Model Optimizer tool with `--reverse_input_channels` argument specified. For more information about the argument, refer to **When to Reverse Input Channels** section of [Converting a Model Using General Conversion Parameters](../../docs/MO_DG/prepare_model/convert_model/Converting_Model_General.md).
 
-### Synchronous API
-
-For synchronous mode, the primary metric is latency. The application creates one infer request and executes the `Infer` method. A number of executions is defined by one of the two values:
-* Number of iterations defined with the `-niter` command-line argument
-* Time duration specified with the `-t` command-line argument
+### Latency and Throughput-focused Inference Modes
+In many cases the primary performance metric is the time (in milliseconds) for an individual inference request.
+For conventional devices the best latency is usually achieved when the application operates single inference request ('-nireq 1').
```

Comment:
It can confuse users because a number of streams is not 1 by default.

Reply:
Good point, so I changed to refer to the legacy "-api sync" instead